components/training/eventBooking.tsx

```diff
@@ -165,8 +166,23 @@ const EventHeader = ({ duration, price, schema }) => {
           <div className="text-xs uppercase text-gray-500">
             {EventModel.PRICE}
           </div>
-          {EventModel.CURRENCY}
-          <span>{price}</span> {EventModel.INCLUDE_GST}
+          <span
+            className={classNames({
+              "text-gray-450 line-through":
+                earlyBirdPrice !== undefined &&
+                earlyBirdPrice !== null &&
+                earlyBirdPrice !== -1,
```

Comment:
Shouldn't be a function that returns this as you're using it somewhere else as well?

Reply:
Done - create function isEmpty